.github/workflows/webpack-jumpstart.yml

```diff
@@ -67,9 +72,10 @@ jobs:
         run: |
           set -ux
           mkdir worktree
-          tar -C worktree -x --exclude '.git*' dist package-lock.json < artifact/webpack-jumpstart.tar
-          git --work-tree worktree add dist package-lock.json
-          git --work-tree worktree commit --quiet -m "Build for ${MERGE_SHA}"
+          tar -C worktree -x --exclude '.git*' dist package-lock.json tree < artifact/webpack-jumpstart.tar
+          echo "${BASE_SHA}" > worktree/merge-base
```

Comment:
I'm a bit confused here -- if we can get the original BASE_SHA in this `on_workflow`, why even bother with building the `tree` file in the `build` job? At this point `BASE_SHA == tree`, no?

Reply:
Sorry, that was a thinko of course -- a tree ID != commit SHA. But still, couldn't this do `git rev-parse` the BASE_SHA to a tree object here?

Reply:
Correct.  That will be used to do some magic in the future with the jumpstart script, possibly via a new `--rebase` argument.  I wanted to leave that for later, though.